commit edb2de0e0d5cbe79b5d78340e39527a3ed8b35ee (refs/changes/75/2175/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-06-30T17:37:53+00:00 (2 years, 3 months ago)
    
    DOCKER-1076 leftover jsprim module loading breaks tests suite
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/test/lib/common.js b/test/lib/common.js
index a45b81c..02d9fa1 100644
--- a/test/lib/common.js
+++ b/test/lib/common.js
@@ -17,7 +17,6 @@ var deepEqual = require('deep-equal');
 var difflet = require('difflet');
 var exec = require('child_process').exec;
 var fmt = require('util').format;
-var jsprim = require('jsprim');
 var libuuid = require('libuuid');
 var VError = require('verror').VError;
 
